fix: backslash in tagged template

Jack-Works opened the pull request:
This PR fixes `\n` being extracted to `\\n`.

timofei-iatsenko commented on src/ast_utils.rs:
```diff
@@ -52,14 +52,22 @@ pub fn get_expr_as_string(val: &Box<Expr>) -> Option<String> {
   }
 }
 
+pub fn get_template_string_cooked_string(element: &TplElement) -> String {
+  if let Some(cooked) = &element.cooked {
+    cooked.to_string()
+  } else {
+    element.raw.to_string()
+  }
+}
```
Could be done as a one-liner 

```rust
tpl.cooked.as_ref().unwrap_or(&tpl.raw).to_string()
```

Also, let's rename a function to 

`get_tpl_cooked_or_raw_string`

timofei-iatsenko reviewed (changes requested):
There two changes in the code, but only one test added. It seems there are cases which is not covered. 

The second case might be

```jsx
<Trans>{`Multiline\nstring`}</Trans>
```
 (please test it with and without fix, because i just took it from mi mind)

Jack-Works commented:
Hi @timofei-iatsenko thanks for the review, I have updated the PR and added one case, but I cannot fix that case. I tried 2 to 3 different ways but to fix that case it looks like I need to rewrite a lot.

Jack-Works commented:
looks like the new major version no longer does anything to the strings in tagged templates, closed for now
https://js-lingui-git-next-crowdin.vercel.app/releases/migration-5#no-whitespace-cleaning-in-js-macros
